app/controllers/publishers/vacancies/application_controller.rb

```diff
@@ -1,7 +1,32 @@
 require "indexing"
 
 class Publishers::Vacancies::ApplicationController < Publishers::BaseController
-  include Publishers::JobCreationHelper
+  def steps_config
```

Comment:
I wonder if this config would prefer to live in `ProcessSteps`; saves various before_actions and passings of arguments. It feels like a constant that belongs to that class.

Reply:
this config is specific to the create vacancy process, a different config will be needed for application

Reply:
I see. Carry on.

Reply:
I don't think any of these should be a `before_action`, rather they can be memoized helpers:

```ruby
def steps_config
  @steps_config ||= { ... }
end
helper_method :steps_config

```